package.json

```diff
@@ -23,6 +23,7 @@
     "postbabelify": "ncc build lib/cli-peer-dependencies.js -o peers",
     "rebuild-fixtures": "npm run babelify && babel-node scripts/rebuildFixtures.js",
     "prepublishOnly": "npm install --force && npm run babelify && babel-node scripts/build.js && node scripts/build-plugins.js",
+    "prepare": "babel-node scripts/create-plugin-list.js",
```

Comment:
We should run this before transpiling the project with babel or it would not be included in the build.

Reply:
This will be included in the build. `prepare` runs on local `npm install` and before `prepublishOnly` (see [docs](https://docs.npmjs.com/cli/v7/using-npm/scripts#life-cycle-scripts)).

Reply:
Yes, but not before `babelify` which should be run before tests😃

Reply:
But the generated file will only change when a new plugin is added. Is that enough to justify generating it each time `babelify` is run? To be fair, it does not take that long.

Reply:
That's a good point, let's wait to hear what the maintainers think.

Reply:
Let's throw it in `prebabelify` for now 👍

Reply:
I'll keep the `prepare` so there are no linter errors when first installing the project (eslint will error if importing a module that does not exist). I cannot find a lifecycle script that runs only after local installation (with no arguments), so unfortunately the script will run twice before publish.